.github/workflows/v0.3-tm2tm.yml

```diff
@@ -14,9 +14,18 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Wait for build to succeed
-        uses: lewagon/wait-on-check-action@e2558238c09778af25867eb5de5a3ce4bbae3dcd
+        uses: lewagon/wait-on-check-action@v1.3.1
+        if: github.event_name != 'pull_request'
```

Comment:
Can you change it to single step like v0.2
```
ref: ${{ (github.event_name == 'pull_request' && github.event.pull_request.head.sha) || github.sha }}
```

Reply:
https://github.com/datachainlab/yui-relayer-build/pull/5/commits/f7877a10ba226ebef0818be97fd7c5c18e795af1
Good code, I learned a lot